e2e/vm/vm_test.go

```diff
@@ -82,3 +84,16 @@ var resetVM = func(o *option.Option, installed bool) string {
 
 	return limaConfigFilePath
 }
+
+var resetDisks = func(o *option.Option, installed bool) {
+	if installed {
+		path, err := exec.LookPath(e2e.InstalledTestSubject)
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+		realFinchPath, err := filepath.EvalSymlinks(path)
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+		gomega.Expect(os.RemoveAll(filepath.Join(realFinchPath, "../../lima/data/_disks")))
+	} else {
+		gomega.Expect(os.RemoveAll("../../_output/lima/data/_disks")).ShouldNot(gomega.HaveOccurred())
+	}
+	gomega.Expect(os.RemoveAll(path.Join(os.Getenv("HOME"), ".finch", ".disks"))).ShouldNot(gomega.HaveOccurred())
```

Comment:
I think this should work for runners but it seems overly broad for local e2e testing as it would delete the installed version's disk as well. Maybe it'd be better to follow the symlink for `../lima/data/_disks/finch/datadisk`?

Reply:
Good point, I'll update it. Would be nice if we had the disk delete command merged in Finch

Reply:
yeah, I can revisit that

Reply:
updated in latest rev